.github/workflows/publish.yml

```diff
@@ -15,7 +15,6 @@ jobs:
   publish:
     name: Publish
     runs-on: ubuntu-latest
-    if: github.repository == 'IABTechLab/uid2-android-sdk' && github.ref == 'refs/heads/main'
```

Comment:
This seemed to be preventing it from running, so since we have `branches: main`, I removed.